packages/codemods/transforms/migrate-grommet-icons-to-hpe.js

```diff
@@ -335,6 +336,15 @@ module.exports = function transformer(file, api, options) {
             );
           }
 
+          if (iconName === 'Share' || iconName === 'share') {
+            console.warn(
+              `⚠️  Warning: Verify if "Share" is the correct usage in this instance.
+              The previous "Share" glyph was often used to symbolize and "external link" opening a new browser window. 
+                 Use "NewWindow" if the usage is communicating an external link.` +
+                `as an alternative if applicable. (${file.path})`,
```

Comment:
Why do we have this part? 

``+ `as an alternative if applicable. (${file.path})` ``